exercise_08/formatStringByWordsLength.js

```diff
@@ -16,3 +16,23 @@
  * CASE #2 * input: "This is short one."
  * output: "this is short one."
  */
+import getTotalWordsFromString from '../exercise_02/getTotalWordsFromString.js';
+
+/**
+ * Converts the case of a given string based on the number of words in it.
+ * If the total number of words is greater than 5, returns the string in uppercase.
+ * Otherwise, returns the string in lowercase.
+ * @param {string} sentence - The string whose number of words will be checked.
+ * @returns {string} The formatted string in either uppercase or lowercase.
+ */
+const formatStringByWordsLength = function (sentence) {
+   let totalWords = getTotalWordsFromString(sentence)
+   
+   if (totalWords > 5) {
```

Comment:
Avoid magic numbers. 

```suggestion
   if (totalWords > THRESHOLD) {
```
- [ ] Extract number to constant